src/MetaboxRegister.php

```diff
@@ -96,90 +95,6 @@ public function register_meta_box_post(): void
         );
     }
 
-    /**
-     * Register Campaign Information meta box.
-     */
-    public function register_meta_box_campaign(): void
-    {
-        $post_types = [ 'page', 'campaign', 'post' ];
-        $analytics_values = AnalyticsValues::from_cache_or_api_or_hardcoded();
-
-        // P4 Datalayer/Campaign fields.
-        $p4_campaign_fields = new_cmb2_box(
-            [
-                'id' => 'p4_campaign_fields',
-                'title' => __('Analytics & Tracking', 'planet4-master-theme-backend'),
-                'object_types' => $post_types, // Post type.
-                'closed' => true, // Keep the metabox closed by default.
-                'context' => 'side', // show cmb2box in right sidebar.
-                'priority' => 'low',
-                'show_names' => false, // Hide the labels.
-            ]
-        );
-
-        $global_project_options = self::maybe_add_current_post_value(
```

Comment:
It seems that `maybe_add_current_post_value` was only used in this function, so we can remove it too 🧹